lib/plugins/aws/deploy/compile/events/apiGateway/lib/permissions.js

```diff
@@ -8,29 +8,6 @@ module.exports = {
     _.forEach(this.serverless.service.functions, (functionObj, functionName) => {
       functionObj.events.forEach(event => {
         if (event.http) {
-          let method;
```

Comment:
Because we give permission to the entire APIG service to invoke this lambda function, so there's no need for it to be unique for each method. I should've done it this way from the beginning.